boards/kaga-fei/ec4l15ba1/ec4l15ba1_nrf54l15_cpuflpr_defconfig

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# Copyright (c) 2025 KAGA FEI Co., Ltd.
```

Comment:
If the file is exactly the same as the `boards/nordic/nrf54l15dk/nrf54l15dk_nrf54l15_cpuflpr_xip_defconfig`, it is better not to add copyright.